tests/upgrades/test_repository.py

```diff
@@ -371,3 +372,152 @@ def test_post_scenario_sync_large_repo(self, target_sat, pre_upgrade_data):
         repo = target_sat.api.Repository(id=rh_repo_id).read()
         res = repo.sync(timeout=4000)
         assert res['result'] == 'success'
+
+
+class TestSimpleContentAccessOnly:
+    """
+    The scenario to test simple content access mode before and after an upgrade to a
+    satellite with simple content access mode only.
+    """
+
+    @pytest.mark.rhel_ver_list([7, 8, 9])
+    @pytest.mark.no_containers
+    @pytest.mark.pre_upgrade
+    def test_pre_simple_content_access_only(
+        self,
+        target_sat,
+        save_test_data,
+        rhel_contenthost,
+        upgrade_entitlement_manifest_org,
+        upgrade_entitlement_manifest,
+    ):
+        """Register host with an activation key that has one custom repository(overriden to enabled)
+        and one Red Hat repository enabled but no subscriptions attached. Assert that only
+        the red hat repository is enabled on the host and that the host is in entitlement mode.
+
+        :id: preupgrade-d6661342-a348-4dd5-9ddf-3fd630b1e286
+
+        :steps:
+            1. Before satellite upgrade
+            2. Create new organization and location
+            3. Upload a manifest in it
+            4. Create a ak with repos and no subscriptions added in it
```

Comment:
which one is correct `added`/`attached`? non-blocking comment
```suggestion
            4. Create a ak with repos and no subscriptions attached in it
```

Reply:
you add subscriptions not attach, but ill fix the grammar

Reply:
okay